### DISCUSSION TITLE
duplicate materials

### DISCUSSION THREAD

**Discussion by bribrioche** [atteneder/glTFast#475] [Q&A]
Hello,
when exporting in GLTF, I noticed that when several objects have the same material, the export saves this same material several times.
is it possible to optimize this and to have in the export, only once a texture used on several objects ?

**Comment by atteneder** [comment]
If that's true, then this is a regression.

Are we talking about the same material (identical Material object) or equal materials (different material objects with same parameters)?

Can you provide a simple test scene that reproduces the *issue*.
